server/constants/emojiTypes.js

```diff
@@ -0,0 +1 @@
+export const EMOJI_TYPES = ['thumbsup', 'thumbsdown', 'smiley', 'tada', 'confused', 'heart', 'rocket', 'eyes'];
```

Comment:
I'd be in favor of using the EMOJIs directly here (ie `EMOJI_TYPES = ['👍️', ...]`)